apps/prairielearn/src/pages/workspace/workspace.html.ts

```diff
@@ -44,49 +44,39 @@ export function Workspace({
       >
         ${RebootModal({ __csrf_token })} ${ResetModal({ __csrf_token })}
 
-        <nav
-          class="navbar navbar-expand-md navbar-dark bg-info align-items-center"
-          style="height:55px"
```

Comment:
Artificially limiting the navbar height could cause text clipping issues if the font size is set sufficiently large.